### PR TITLE
Find problems in infra after csp-security-policies v1.2.0

### PR DESCRIPTION
**Pull request by yashtewari**
Includes major refactors to the new audit design.

### CODE REVIEW

**Review by eyalkraft — APPROVED** [review]

**Review by DaveSys911 — APPROVED** [review]
Works as a quick fix.

**Review by oren-zohar — CHANGES_REQUESTED** [review]
please include https://github.com/elastic/csp-security-policies/commit/67be1dcd34628b48286d5a7aef76c8da22cec87d in this PR

File: tests/commonlib/utils.py [hunk 1]
```diff
@@ -27,6 +27,8 @@ def get_ES_evaluation(elastic_client, timeout, rule_tag, exec_timestamp,
             print(e)
             continue
 
+        print('MATCHING EVENTS:', len(events))
+
```
Comment (oren-zohar) [inline tests/commonlib/utils.py:31]: left on purpose?

File: tests/commonlib/utils.py [hunk 2]
```diff
@@ -39,6 +41,7 @@ def get_ES_evaluation(elastic_client, timeout, rule_tag, exec_timestamp,
                 continue
 
             if resource_identifier(resource):
+                print('FINDING MATCH:', event)
```
Comment (oren-zohar) [inline tests/commonlib/utils.py:44]: left on purpose?

**Review by oren-zohar — CHANGES_REQUESTED** [review]

**Pull request closed without merging**